apps/webservice/package.json

```diff
@@ -29,9 +30,8 @@
     "@octokit/auth-app": "^7.1.0",
     "@octokit/rest": "catalog:",
     "@octokit/webhooks-types": "^7.5.1",
-    "@opentelemetry/exporter-trace-otlp-proto": "^0.53.0",
-    "@opentelemetry/instrumentation-http": "^0.53.0",
-    "@opentelemetry/instrumentation-pg": "^0.44.0",
+    "@opentelemetry/auto-instrumentations-node": "^0.50.0",
+    "@opentelemetry/exporter-trace-otlp-http": "^0.53.0",
```

Comment:
I think using this will fix the DD issue because we had the exporter as otlp-proto 🤦